docs/source/distributed_training/performance_tuning.md

```diff
@@ -0,0 +1,76 @@
+# Performance Tuning
+
+Getting the best performance from your training jobs is of utmost importance -- GPUs are expensive! Streaming's default parameters give great performance out-of-the-box for most model configurations. Performance with Streaming mainly deals with downloading and storing shards optimally.
+
+Great performance with Streaming means that dataloading is never a bottleneck during model training. Streaming also provides the [Streaming Simulator tool](#streaming-simulator) to help with performance optimization. Let's dive in!
+
+## Downloading shards
+
+Streaming downloads dataset shards on the fly to make sure that the samples they contain are ready to be trained with. Refer to this [section](../dataset_configuration/shard_retrieval.md#controlling-shard-downloads) for information on how to control shard downloads. Some potential issues we have seen:
+- The dataset `predownload` and dataloader `workers` are set too low. Either increase the `predownload` parameter to StreamingDataset, or increase the number of workers for your dataloader, to allow more samples to be prepared ahead of time.
+- Shard downloads are not balanced, often in conjunction with low download bandwidth. To determine if shuffling is increasing download demand too much, try running with `shuffle` set to `False`. Then, make sure `shuffle_algo` is set to `'py1e'` (more info [here](../dataset_configuration/shuffling.md#py1e-default)) to help balance out inter- and intra-node shard downloads while maintaining shuffle quality. If this is still slowing down training, try the [`'py1s'`](../dataset_configuration/shuffling.md#py1s) or [`'py2s'`](../dataset_configuration/shuffling.md#py2s) shuffling algorithms.
+
+## Storing shards
+
+Once shards are downloaded, they are stored on each node's disk, and are available to that node's GPUs. Refer to this [section](../dataset_configuration/shard_retrieval.md#controlling-shard-downloads) for information on controlling how shards are stored. The main issue that can crop up here is when the node's available disk space is less than the cache size required to store dataset shards. If `cache_limit` is not set, each node's cache size for shards is given by:
+
+$$L = \frac{S \cdot N}{P}$$
```

Comment:
nit: Looks like the Math are not getting rendered correctly in the doc :(

Reply:
fixed!